sandbox/snippets/101functions/httpTriggers/accessing-querystrings.md

```diff
@@ -0,0 +1,38 @@
+## Accessing querystring values in Http Triggers
+
+When making HTTP request, you have the ability of supplying addional parameters via a query string to the request URL which can be retrieved from by the website or web api to alter how the response is returned.
```

Comment:
- additional
- Should it be "query string" or "querystring" (in heading)?
- API (caps)